### DISCUSSION TITLE
Datbase Logs?

### DISCUSSION THREAD

**Discussion by jordanparker6** [planetscale/discussion#437] [Q&A] [answered]
Is it possible to get a stream or a batch export of the database transaction logs from planet scale?

It would be great to store the transaction logs so I can rebuild the dataset from any point and get a full record of all inserts to build analytics datasets. It would be nice to just use planet scale as an operational database and have all the transaction logs sent to storage and analysed with something like BigQuery or Athena.

**Comment by fooforge** [comment, marked as answer]
@jordanparker6 Hey 👋,

We currently do not expose MySQL's binary logs from PlanetScale databases below our Enterprise offerings. It is something that, I think, is worth exploring for us to implement eventually, but it's not on our immediate roadmap.

We do have [Airbyte](https://planetscale.com/docs/integrations/airbyte) and [Stitch](https://planetscale.com/docs/integrations/stitch) integrations if you need to sync your data to another location, however.
